### PR TITLE
Bugfix/web 566

### PR DESCRIPTION
**Pull request by jasmeet17**
- Credit card expiry date supports 2 last digit of the year
- Credit card expiry date field works with auto-fill details
- Updated Readme
- Added 1 testcase

### CODE REVIEW

File: assets/js/validator.js [hunk 1]
```diff
@@ -114,8 +114,29 @@
...
+
+            if (arr.length > 1){
+                var tempYear = arr[1];
+                tempYear = tempYear.trim();
+
+                if (tempYear.length >= 4){
+                    tempYear = tempYear.substring(tempYear.length -2);
+                    str = arr[0] + '/' + tempYear;
+
+                    return str;
```
Comment (e-wipond) [inline assets/js/validator.js]: This return is redundant.

File: payfields/tests/karma.conf.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ module.exports = function(config) {
         browsers: ['Chrome'],
         frameworks: ['jasmine'],
         files: [
-          '../../build/payfields/beanstream_payfields.js',
+          '../../build/v0.0.0/payfields/beanstream_payfields.js',
```
Comment (e-wipond) [inline payfields/tests/karma.conf.js]: In other files you have a version of /v1.1.0/

Comment (jasmeet17) [inline payfields/tests/karma.conf.js]: this is for testing at local. So it will be v0.0.0

**Comment by Lmasniuk** [comment]
Feature branches should not merge into Master. Please re-make this pull request to pull from web-566 into Dev branch.

**Pull request closed without merging**